src/plugins/graph/imports/components/axis/models/axis-model.ts

```diff
@@ -81,6 +81,12 @@ export const NumericAxisModel = AxisModel
   }))
   .actions(self => ({
     setDomain(min: number, max: number) {
+      // Clean NaNs since they will make the document crash
+      if (!isFinite(min) || !isFinite(max)) {
+        console.warn('NumericAxisModel.setDomain called with non-numeric arg(s): ', min, max);
```

Comment:
```suggestion
        console.warn('NumericAxisModel.setDomain called with non-numeric arg(s):', min, max);
```
FYI: console methods automatically add spaces between arguments, so you don't need to add your own.